content/en/docs/reference/using-api/cel.md

```diff
@@ -0,0 +1,330 @@
+
+In additional to the CEL community libraries, Kubernetes includes CEL libraries
+that are available everywhere CEL is used in Kubernetes.
+
+### Kubernetes List Library
+
+The list library includes `indexOf` and `lastIndexOf`, which work similar to the
+strings functions of the same names. These functions either the first or last
+positional index of the provided element in the list.
+
+The list library also includes `min`, `max` and `sum`. Sum is supported on all
+number types as well as the duration type. Min and max are supported on all
+comparable types.
+
+`isSorted` is also provided as a convenience function and is supported on all
+comparable types.
+
+Examples:
+
+| CEL Expression                                                                     | Purpose                                                   |
+|------------------------------------------------------------------------------------|-----------------------------------------------------------|
+| `names.isSorted()`                                                                 | Verify that a list of names is kept in alphabetical order |
+| `items.map(x, x.weight).sum() == 1.0`                                              | Verify that the "weights" of a list of objects sum to 1.0 |
+| `lowPriorities.map(x, x.priority).max() < highPriorities.map(x, x.priority).min()` | Verify that two sets of priorities do not overlap         |
+| `names.indexOf('should-be-first') == 1`                                            | Require that the first name in a list if a specific value | 
+
+See the [Kubernetes List Library](https://pkg.go.dev/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/library#Lists)
+godoc for more information.
+
+### Kubernetes Regex Library
+
+In addition to the `matches` function provided by the CEL standard library, the
+regex library provides `find` and `findAll`, enabling a much wider range of
+regex operations.
+
+Examples:
+
+| CEL Expression                                              | Purpose                                                  |
+|-------------------------------------------------------------|----------------------------------------------------------|
+| `"abc 123".find('[0-9]*')`                                  | Find the first number in a string                        |
+| `"1, 2, 3, 4".findAll('[0-9]*').map(x, int(x)).sum() < 100` | Verify that the numbers in a string sum to less than 100 |
+
+See the [Kubernetes Regex Library](https://pkg.go.dev/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/library#Regex)
+godoc for more information.
+
+### Kubernetes URL Library
+
+To make it easier and safer to process URLs, the following functions have been added:
+
+- `isURL(string)` - Check if a string is a valid URL according to the [Go's
+  net/url](https://pkg.go.dev/net/url#URL) package. The string must be an
+  absolute URL.
+- `url(string) URL` Converts a string to a URL or results in an error if the
```

Comment:
nit: `url(string) URL` Converts -> `url(string) URL - Convert` to keep style same as above `isURL(string)`.
Could also update `isURL(string) - Check` to `isURL(string) checks`

Reply:
Good catch. I resolved this the otherway.